### PR TITLE
Force refresh of projects when OpenShift Origin clicked

### LINKED ISSUE #1733: "cache" issues in web console cause weird behaviors
I have noticed a lot of issues with the console that I can only attribute to "caching". What I mean by this is that, for example, if I click the "OpenShift Origin" logo at the top, I am taken to the project list, but it displays old information.

It seems that to force an actual reload of a page I can't click anything. I have to use F5 or re-enter the URL in the browser and hit enter.

I can't really nail down all the instances where this happens, but I'll try to capture them in this issue.

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #1733

### CODE REVIEW

**Comment by spadgett** [comment]
@liggitt please review

File: assets/app/scripts/controllers/projects.js [hunk 1]
```diff
@@ -8,12 +8,23 @@
...
  */
 angular.module('openshiftConsole')
-  .controller('ProjectsController', function ($scope, $location, DataService, AuthService, Logger) {
+  .controller('ProjectsController', function ($scope, $location, $route, DataService, AuthService, Logger) {
     $scope.projects = {};
     $scope.alerts = $scope.alerts || {};
     $scope.emptyMessage = "Loading...";
     $scope.canCreate = undefined;
 
+    var logoClickHandler = function(e) {
```
Comment (liggitt) [inline assets/app/scripts/controllers/projects.js]: good place to use namespaced events:
```
    $('#openshift-logo').on('projectsPage.click', function(e) {
      // Force a reload. Angular doesn't reload the view when the URL doesn't change.
      $route.reload();
    });
    $scope.$on('$destroy', function(){
      // The click handler is only necessary on the projects page.
      $('#openshift-logo').off('projectsPage.click');
    });
```

Comment (spadgett) [inline assets/app/scripts/controllers/projects.js]: I'll make the update.

**Comment by spadgett** [comment]
@liggitt I added the namespaced event.

**Comment by liggitt** [comment]
LGTM, [merge]

**Comment by spadgett** [comment]
@liggitt I fixed the test failure by mocking $route.reload()

**Comment by liggitt** [comment]
why was that getting triggered? that's a little concerning

**Comment by liggitt** [comment]
superseded by #2514

**Pull request closed without merging**